hveto/const.py

```diff
@@ -0,0 +1,33 @@
+import os
```

Comment:
I think this is the first time you've contributed a brand-new file, right? This is a cool milestone 🥰 

There is some metadata that should go into commented-out lines at the top of this file before the imports, information about license and copyright, see other files in the repo for the template (just replace the name with your name and the year with this year). There should also be an `__author__` variable after the imports, as in the other files.

Reply:
That's true, a brand new file :)
Thanks.